katdal/applycal.py

```diff
@@ -302,6 +337,8 @@ def add_applycal_sensors(cache, attrs, data_freqs, cal_stream, cal_substreams=No
                        "spectral attributes", cal_stream)
         return
     targets = cache.get('Observation/target')
+    if gaincal_fluxes is None:
+        gaincal_fluxes = attrs.get('measured_flux', {})
```

Comment:
You might want to merge the user provided dict with the cal dict- so that someone can change the flux of just one in a long list of calibrators without deleting all of the others.

Reply:
A side effect of this is that `gaincal_fluxes=None` now disables flux calibration, while the default becomes `gaincal_fluxes={}` instead.